src/components/AttractionCard/AttractionCard.js

```diff
@@ -1,8 +1,25 @@
 import styled from 'styled-components/macro'
+import { ReactComponent as LikeSVG } from '../../icons/liked.svg'
+import { ReactComponent as UnlikeSVG } from '../../icons/unliked.svg'
 
-export default function AttractionCard({ name, image }) {
+const like = <LikeSVG />
+const unlike = <UnlikeSVG />
+
+export default function AttractionCard({
+  name,
+  image,
+  likePlace,
+  likedPlaces,
+}) {
   return (
     <Card key={name}>
+      <LikeButton
+        role="button"
+        aria-label="toggle-like"
+        onClick={() => likePlace(name)}
```

Comment:
In your `App.js` you give `likePlace` a parameter which is called `currentId`, here you give the name. Do you plan to have an id for these objects? It would be better to be consistent. If you expect `currentId`, you should give the function an id.

Reply:
I was considering an ID first, but then changed my mind and forgot to adjust the function. Thx 👍🏻